### ISSUE TITLE
XDG spec for config file location

### ISSUE THREAD

**Issue by cl3wis** [mjpost/bibsearch#35]
(from the guidelines [here](https://specifications.freedesktop.org/basedir-spec/latest/ar01s03.html))

The config could be looked up at these standard locations, in order:

1) `$XDG_CONFIG_HOME/bibsearch/config`
2) `$HOME/.config/bibsearch/config`
3) (current) `$HOME/.bibsearch/config`

Similarly, the `lastSearch.yaml`, `bib.db`, and `download_dir` could live in `$XDG_DATA_HOME` and/or `$XDG_CACHE_HOME`.

Thoughts? If the file lookups are done preferentially, it shouldn't break any existing setups.

**Comment by mjpost** [comment]
I've never heard of this spec; which is not to say that it is wrong, only that this surprises me. If this is a standard I'd be happy to see it in bibsearch. Thoughts, @davvil?

**Comment by davvil** [comment]
Yes, that's actually the right way to do it. I'll have a go at it.

**Comment by cl3wis** [comment]
I'm open to work on this as well.

**Comment by mjpost** [comment]
You should do it. It’d be nice to have new contributors.

**Comment by cl3wis** [comment]
Sure, I'll tackle this.


How is (is there any) Windows support for bibsearch?

**Comment by mjpost** [comment]
I have not tried it on Windows.
